tests/integration_tests/clouds.py

```diff
@@ -25,6 +26,65 @@
 log = logging.getLogger('integration_testing')
 
 
+def _get_ubuntu_series() -> list:
+    """Use distro-info-data's ubuntu.csv to get a list of Ubuntu series"""
+    try:
+        with open("/usr/share/distro-info/ubuntu.csv") as f:
+            reader = csv.DictReader(f)
+            return [row["series"] for row in reader]
+    except FileNotFoundError:
+        log.info(
+            "distro-info-data must be installed to guess Ubuntu os/release"
```

Comment:
If you end up using distro-info directly, we'd change this
```suggestion
            "distro-info must be installed to guess Ubuntu os/release"
```